src/applications/facility-locator/tests/mapbox/mapbox-api-services.unit.spec.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+
+import MapboxClient from '@mapbox/mapbox-sdk';
+
+import { mapboxToken } from '../../utils/mapboxToken';
+
+const mapboxClient = MapboxClient({ accessToken: mapboxToken });
+
+const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
+
+const mbxClient = mbxGeo(mapboxClient);
+
+import { BOUNDING_RADIUS } from '../../constants';
```

Comment:
This is a somewhat trivial ask, but could we group the `import` statements together for readbility? Perhaps something like this:

```js
// Node modules.
import { expect } from 'chai';
import MapboxClient from '@mapbox/mapbox-sdk';
import mbxGeo from '@mapbox/mapbox-sdk/services/geocoding';
// Relative imports.
import { BOUNDING_RADIUS } from '../../constants';
import { mapboxToken } from '../../utils/mapboxToken';


const mapboxClient = MapboxClient({ accessToken: mapboxToken });
const mbxClient = mbxGeo(mapboxClient);
```